notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb

```diff
@@ -0,0 +1,2824 @@
+{
```

Comment:
<p>combine the 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>done</p>